recipes/ensmallen/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "0.8.96" %}
+
+package:
+  name: ensmallen_graph
```

Comment:
@dillonroach, @conda-forge/staged-recipes, I named this `ensmallen_graph` because there's an unrelated https://github.com/conda-forge/ensmallen-feedstock